api/models.py

```diff
@@ -647,7 +647,7 @@ class Meta:
         verbose_name_plural = _('region snippets')
 
     def __str__(self):
-        return self.snippet
+        return (self.snippet[:80] if self.snippet else None) or (self.image.url if getattr(self, 'image') else '')
```

Comment:
Without default value in `getattr`, it throws AttributeError error
```suggestion
        return (
            self.snippet[:80] if self.snippet else None
        ) or (
            self.image.url
            if getattr(self, 'image', None) and self.image.url
            else ''
        )
```